app/views/settings/index.html.erb

```diff
@@ -27,7 +27,11 @@ first_category = ordered_settings.values.flatten.first.try(:category)
           <% setting.each do |item| %>
             <tr>
               <td ><%= translate_full_name(item) %></td>
-              <td class="setting_value"><%= value(item) %></td>
+              <% if ((item.value != item.default) && !(item.default.nil?)) %>
+                  <td class="setting_value"><strong><%= value(item) %></strong></td>
+              <% else %>
+                  <td class="setting_value"><%= value(item) %></td>
+              <% end %>
```

Comment:
This could leverage `has_default?` method from https://github.com/theforeman/foreman/pull/6089

Reply:
@lzap Yes, sure. Will update the changes once your PR is merged :)

Reply:
It's now merged!

Reply:
Updated the changes!